Automattic/pocket-casts-android
# Support RTL notes in the webview

## Description
Added RTL support to the notes that display in the webview of the EpisodeFragment and NotesFragment. `dir="rtl"` needs to set on the notes html for it to display correctly so check if the device locale is RTL and adding the appropriate tag to the html body.

Fixes #109 

## Testing Instructions
1. Set the device language to RTL language
2. Open podcast episode
3. See that the notes in the webview are aligned RTL

## Screenshots or Screencast 
![Screenshot_20221030_050715](https://user-images.githubusercontent.com/22520267/198863521-d31f8e7d-b1f9-4bad-96e7-7e2075671c6e.png)
![Screenshot_20221030_050638](https://user-images.githubusercontent.com/22520267/198863523-0fae374a-dd56-4d85-86ab-95f4f8fd2470.png)

## Checklist
 
#### I have tested any UI changes...
